NetKAN/TrizzleAeroSpaceCompany.netkan

```diff
@@ -0,0 +1,16 @@
+{
+    "spec_version": "v1.10",
+    "$kref": "#/ckan/kerbalstuff/887",
+    "identifier"  : "TrizzleAeroSpaceCompany",
+    "license"     : "MIT",
+    "resources": {
+        "homepage": "https://kerbalstuff.com/mod/887/Trizzle%20AeroSpace%20Company"
+    },
+    "install": [
+        {
+            "find_regexp": "TrizzleAeroSpace_v\\d\\d*\\.\\d\\d*",
```

Comment:
The shorthand for "one or more" is `+`, i.e. `TrizzleAeroSpace_v\\d+\\.\\d+`, but this is fine as is.